PhysicsTools/SelectorUtils/interface/VersionedIdProducer.h

```diff
@@ -118,6 +121,8 @@ VersionedIdProducer(const edm::ParameterSet& iConfig) {
     produces<edm::ValueMap<float> >(idname); // for PAT
     produces<edm::ValueMap<unsigned> >(idname);  
     produces<edm::ValueMap<unsigned> >(idname+std::string(bitmap_label));
+    produces<edm::ValueMap<vid::CutFlowResult> >(idname);
+    //produces<UserDataCollection>(idname);
```

Comment:
is this commented out UserDataCollection needed for upcoming development?
If not, please remove